pdf_gui.py

```diff
@@ -70,13 +70,20 @@ def __init__(self, master):
         ### END OF TO-DO. ###
         
         # Canvas creation to display PDF document pages.
-        self.canvas = Canvas(self.master, bg="white")
+        self.canvas = Canvas(self.master, width=854, height=480, bg="white")
         self.canvas.pack(side=LEFT, fill=BOTH, expand=True)
+        self.display_page(0, pdf_file)
```

Comment:
Again, this is fine since we are just testing the display page function when running the Python file. However, we will have to remove this once we submit the link to the Github repository (along with slides and report) when the project is due.